src/pages/customer/CustomerReservationDetailModal.module.scss

```
@@ -21,8 +21,8 @@
       border-radius: 50%;
       position: absolute;
       z-index: 4;
-      bottom: 20px;
-      right: 20px;
+      bottom: 14px;
+      right: 25px;
```

object-fit: cover 를 포함하면 이미지 비율을 유지할 수 있을거 같아요!
```suggestion
      right: 25px;
      object-fit: cover;
```